source/chapters/user_interface.rst

```diff
@@ -381,32 +399,29 @@ Transport Controls
    RJ Ryan <rryan@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
 
-.. figure:: ../_static/Mixxx-200-Latenight-Deck-Transport.png
+.. figure:: ../_static/2.2/transport-controls.png
   :align: center
-  :width: 55px
+  :width: 80px
   :figwidth: 100%
   :alt: The transport controls of the deck
   :figclass: pretty-figures
 
   The transport controls of the deck
 
-**Fast-Rewind button**
- As long as the button is pressed, the track will play in reverse with
- increased speed. Right-clicking on the button will seek the play position to
- the beginning of the track.
+**Beatjump/ Loop Move Size**
+  You can type or use the arrow keys in the input box to select the number of beats to jump or move, with the Beatjump Forward/ Backward buttons.
 
-**Fast-Forward button**
- As long as the button is pressed, the track will play with increased speed.
- Right-clicking on the button will seek the play position to the end of the
- track.
+**Beatjump Forward button**
+  Left-clicking this button while the loop is inactive will make the track jump forward by the set number of beats.
+  Left-clicking this button while the loop is active will move the loop forward by the set number of beats.
+  Right-clicking this button while the loop is inactive will make the track jump forward by one beat.
+  Right-clicking this button while the loop is active will move the loop forward by one beat.
```

Comment:
I know we split it up like this in the tooltips, but for the text I find it easier to group by loop active/inactive.
So your addition is correct, though I think it's easier to grasp when written like this:
```suggestion
  Clicking this while the loop is inactive will make the track jump forward by the set number of beats (left click) or by one beat (right click).
  Clicking this while the loop is active will move the loop forward by the set number of beats (left click) or by one beat (right click).
```
Agree?

Reply:
Also most of our users live with a left-to-right language, so we should mention the *Beatjump Backward button* first?

Reply:
Agreed. This format is easier on the eyes.  Less text and straight to the point.